velox/docs/functions/spark/string.rst

```diff
@@ -212,22 +212,24 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT soundex('Miller'); -- "M460"
 
-.. spark:function:: split(string, delimiter) -> array(string)
-
-    Splits ``string`` on ``delimiter`` and returns an array. ::
+.. spark:function:: split(string, delimiter[, limit]) -> array(string)
+    Splits ``string`` around occurrences that match ``delimiter`` and returns an array with a length of
```

Comment:
@gaoyangxiaozhu Would you generate the docs and verify they render nicely?

It might be helpful to add some examples, especially for cases when delimiter is empty.

Reply:
there are example for empty delimiter - https://github.com/gayangya/velox/blob/gayangya/split_refactor/velox/docs/functions/spark/string.rst?plain=1#L231-L234